charts/fybrik/files/taxonomy/datacatalog.json

```diff
@@ -43,6 +43,34 @@
         }
       }
     },
+    "DeleteAssetRequest": {
+      "type": "object",
+      "required": [
+        "destinationCatalogID"
+      ],
+      "properties": {
+        "destinationAssetID": {
+          "description": "Asset ID of the to-be deleted asset",
+          "type": "string"
+        },
+        "destinationCatalogID": {
+          "description": "The destination catalog id in which the asset to be deleted exists",
```

Comment:
simplify: "... in which the asset exists"

Reply:
Fixed. Thanks.